sdk/notificationhubs/notification-hubs/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "@azure/notification-hubs",
+  "version": "1.0.0.0",
+  "description": "Azure Notification Hubs SDK for JavaScript",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "browser": {},
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/constants.ts",
+        "prefix": "SDK_VERSION"
+      }
+    ]
+  },
+  "types": "types/latest/notification-hubs.d.ts",
+  "typesVersions": {
+    "<3.6": {
+      "*": [
+        "types/3.1/notification-hubs.d.ts"
+      ]
+    }
+  },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:samples": "echo Obsolete",
+    "build:test": "npm run clean && tsc -p . && dev-tool run bundle",
+    "build:types": "downlevel-dts types/latest/ types/3.1/",
+    "build": "npm run build:test && api-extractor run --local && npm run build:types",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* temp types *.tgz *.log",
+    "execute:samples": "dev-tool samples run samples-dev",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "generate:client": "autorest --typescript ./swagger/README.md",
```

Comment:
are we actually using autorest for this one? I don't see any generated client

Reply:
No, there is no autorest involved as it didn't give me a proper result.  Just copy/pasted from template but will fix